addon/run-task.ts

```diff
@@ -182,14 +182,14 @@ export function scheduleTask(
 export function throttleTask(
   obj: EmberObject,
   taskName: any,
```

Comment:
This should be typed as a `string | symbol` (using `any` infers we also allow a function, which we definitely don't)

Reply:
All feedback addressed!